service/src/main/java/bio/terra/drshub/services/DrsResolutionService.java

```diff
@@ -220,11 +222,10 @@ private DrsObject fetchObjectInfo(
     var sendMetadataAuth = drsProvider.isMetadataAuth();
 
     var objectId = getObjectId(uriComponents);
-    log.info(
-        "Requesting DRS metadata for {} with auth required {} from host {}",
-        drsUri,
-        sendMetadataAuth,
-        uriComponents.getHost());
+    String drsRequestLogMessage =
```

Comment:
Is there a reason not to in-line this? Just stylistic?

Reply:
Great question, I defined it for reuse below in the cases where we run into issues with passports.